src/pybind/mgr/dashboard/frontend/src/app/ceph/pool/pool-form/pool-form.component.html

```diff
@@ -325,6 +325,10 @@
                             [ngClass]="{'active': data.crushInfo}"
                             id="crush-info-button"
                             type="button"
+                            ngbTooltip="Placement and
+                            replication strategies or distribution policies that allows to
```

Comment:
Since "strategies" and "policies" are plural, it should be "allow":
```suggestion
                            replication strategies or distribution policies that allow to
```